instances/atlas/brainAtlas/MarsAtlas.jsonld

```diff
@@ -0,0 +1,190 @@
+{
+      "@id": "https://openminds.ebrains.eu/instances/brainAtlas/MarsAtlas",
+      "@type": "https://openminds.ebrains.eu/SANDS/BrainAtlas",
+      "author": [
+            {
+                  "@id": "https://openminds.ebrains.eu/instances/person/auziasGuillaume"
+            },
+            {
+                  "@id": "https://openminds.ebrains.eu/instances/person/brovelliAndrea"
+            },
+            {
+                  "@id": "https://openminds.ebrains.eu/instances/person/coulonOlivier"
+            }
+      ],
+      "custodian": [
+            {
+                  "@id": "https://openminds.ebrains.eu/instances/person/brovelliAndrea"
+            }
+      ],
+      "description": "MarsAtlas is a model of cortical parcellation. It can be applied to any cortical surface via the HipHop parameterization pipeline, available in the BrainVisa Cortical Surface Toolbox, under the Anatomy category.",
+      "digitalIdentifier": [
+            {
+                  "@id": "https://openminds.ebrains.eu/instances/digitalIdentifier/DOI_Mars_cortex_10.1002.hbm.23121"
+            }
+      ],
+      "fullName": "MarsAtlas",
+      "hasTerminology": {
+            "@type": "https://openminds.ebrains.eu/sands/ParcellationTerminology",
+            "definedIn": null,
+            "hasEntity": [
+                  {
+                        "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/VCcm"
```

Comment:
The entities "@id" should always start with the abbreviation of the atlas, e.g.: 
"@id": "https://openminds.ebrains.eu/instances/parcellationEntity/Mars_VCcm"

Reply:
(I think the instances for the PEs actually have the correct "@id") only the here in the atlas it is wrong)

Reply:
fixed